### ISSUE TITLE
Suggestion : Raw label

### ISSUE THREAD

**Issue by Nuranto** [leantony/laravel-grid#53]
Hi,

I think it is not possible to add a label containing html (some fa icon for example). We can do it in rows, using raw parameter, but not in labels, right ?
Well, that's a suggestion ! 

( Thanks for this project...! )

**Comment by leantony** [comment]
Yes, great idea - I'll add this on the next release (2.0.6). Currently, only the content can be set as raw html.

**Comment by Nuranto** [comment]
Thanks! 
I have a lot more if you want :D

**Comment by leantony** [comment]
Yeah, keep em coming
